### ISSUE TITLE
Bad representation of long float values

### ISSUE THREAD

**Issue by t0neg** [zertrin/clikraken#6]
Hi,

I think float values should be explicitly converted to string before output.

Command: 
```
clikraken t -p XRPXBT
```
Output:
```
pair        last       high        low         vol        wavg  vol value          ask        bid
------  --------  ---------  ---------  ----------  ----------  -----------  ---------  ---------
XRPXBT  4.45e-05  4.455e-05  3.684e-05  5.8286e+07  3.9762e-05  2318 XBT     4.386e-05  4.373e-05
```

**Comment by zertrin** [comment]
~~Yeah indeed, but in fact that's something for tabulate upstream to fix/add a knob to set~~

Ok in fact that's doable by passing the `floatfmt` argument to `tabulate()`. The default is `'g'` which as per reading the documentation https://docs.python.org/3/library/string.html#format-specification-mini-language is equivalent to `'.6g'`. I will increase it to `'.12g'` and test a bit before releasing this fix.

**Comment by zertrin** [comment]
while I have now:

```
pair          last      high         low            vol         wavg  vol value           ask         bid
------  ----------  --------  ----------  -------------  -----------  -----------  ----------  ----------
XRPXBT  0.00012533  0.000143  0.00010165  48862496.5805  0.000106659  5212 XBT     0.00012532  0.00012503
```

which looks better,

on the other side I get:

```
pair          last        high         low             vol        wavg  vol value           ask         bid
------  ----------  ----------  ----------  --------------  ----------  -----------  ----------  ----------
ETHEUR   161.72124   175.39846   139.5      233770.548185    158.76435  37 MEUR       161.96999   161.52217
ETHUSD   180         197.98001   160.09114  121007.062642    178.59748  22 MUSD       179.98899   179.95097
XBTEUR  2161.082    2346        1932.08      38841.8682236  2136.13876  83 MEUR      2161.084    2158.484
XBTUSD  2407        2649.996    2201         12812.0302451  2404.45064  31 MUSD      2406.999    2403.491
```

which is a bit too much precision for the column `vol`...

**Comment by zertrin** [comment]
I pushed the change on master in commit 54dd273c for now. Will decide later if it stays like this or if I do something for columns that get too detailed with this change.

**Comment by zertrin** [comment]
I released version 0.6.1 with this change. For now I'm not planning of changing the behaviour so I will close this issue.
